### ISSUE TITLE
Current values are incorrect: need values for INA219 setup

### ISSUE THREAD

**Issue by bentprong** [bentprong/ocp_xavier#1]
Default values don't work as tested on the test pads.   Here are the defines currently used, adopted from the Arduino INA219 library example code.
// INA219 defines
#define U2_SHUNT_MAX_V  0.04      /* Rated max for our shunt is 75mv for 50 A current: */
                                  /* we will measure only up to 20A so max is about 75mV*20/50 */
#define U2_BUS_MAX_V    16.0      /* with 12v lead acid battery this should be enough*/
#define U2_MAX_CURRENT  3.0       /* In our case this is enaugh even tho shunt is capable to 50 A*/
#define U3_SHUNT_MAX_V  0.04      /* Rated max for our shunt is 75mv for 50 A current: */
                                  /* we will mesaure only up to 20A so max is about 75mV*20/50 */
#define U3_BUS_MAX_V    16.0      /* with 12v lead acid battery this should be enough*/
#define U3_MAX_CURRENT  3.0       /* In our case this is enaugh even tho shunt is capable to 50 A*/
#define SHUNT_R         0.01      /* Shunt resistor in ohms (R211 and R210 are the same ohms) */

**Comment by bentprong** [comment]
[ina219.pdf](https://github.com/bentprong/ocp_xavier/files/10758239/ina219.pdf)

**Comment by rickbrandwein** [comment]
Version 1.0.6 appears to be accurate.  With a benchtop supply I am able to make voltage adjustments and see the voltage measurements track.  Next I had pulled the shunt resistors to put an ammeter in series.  Then compared the measurements with the Xavier current measurements prior to removing the shunts.  Verified the 6-8mA reading on the 3.3v line that the card was reporting and the 170-210mA reading on the 12V line.
